x-pack/plugins/monitoring/public/components/table/table.js

```diff
@@ -123,10 +124,12 @@ class MonitoringTableUI extends React.Component {
         break;
       default:
         throw new Error(
-          this.props.intl.formatMessage({
-            id: 'xpack.monitoring.table.unknownTableActionTypeErrorMessage',
-            defaultMessage: `Unknown table action type {action}! This shouldn't happen!` }, {
-            action
+          i18n.translate('xpack.monitoring.table.unknownTableActionTypeErrorMessage', {
+            defaultMessage: 'Unknown table action type {action}! This shouldn\'t happen!',
```

Comment:
Let's wrap the string into back-tips instead of using backslash:
```suggestion
            defaultMessage: `Unknown table action type {action}! This shouldn't happen!`,
```